<?xml version="1.0" encoding="utf-8"?>

<launch>

  <!-- Load Parameter -->
  <param name="use_cpu"           value="false" />

  <!-- Start yolov5 and ros wrapper -->
  <node pkg="yolov5_ros" type="yolo_v5.py" name="yolov5_ros" output="screen" >
    <param name="weight_path"       value="yolov5s.pt"/>
    <param name="pub_topic"         value="/yolov5/BoundingBoxes" />
    <param name="camera_frame"      value="camera_color_frame"/>
    <param name="conf"              value="0.6" />
    <param name="iou"              value="0.3" />
  </node>
  <node pkg="yolov5_ros" type="detect_show.py" name="detect_show" output="screen" >
  </node>
</launch>
